autoload/easy_align.vim

```diff
@@ -89,9 +89,33 @@ function! s:floor2(v)
   return a:v % 2 == 0 ? a:v : a:v - 1
 endfunction
 
+function! s:get_highlight_group_name(line, col)
+  let l:hl = synIDattr(synID(a:line, a:col, 0), 'name')
```

Comment:
I don't use `l:` prefix for local variables.

Reply:
Updated, please review